scripts/install_avalanchego_release.sh

```diff
@@ -27,22 +27,72 @@ if [[ ${GOOS} == "darwin" ]]; then
   AVAGO_DOWNLOAD_PATH=${BASEDIR}/avalanchego-macos-${VERSION}.zip
 fi
 
-AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH:-${BASEDIR}/avalanchego-${VERSION}}
+# TODO: Please read:
+# not sure why the version isn't included in CI currently... open to suggestions on how to handle this better
+AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH:-${BASEDIR}/avalanchego}
+VERSIONED_AVALANCHEGO_BUILD_PATH=${AVALANCHEGO_BUILD_PATH}-${VERSION}
 mkdir -p $AVALANCHEGO_BUILD_PATH
+mkdir -p $VERSIONED_AVALANCHEGO_BUILD_PATH
 
 if [[ ! -f ${AVAGO_DOWNLOAD_PATH} ]]; then
```

Comment:
You're absolutely right for commits and binary-releases, but this doesn't apply for branches, which was specified in the [original issue](https://github.com/ava-labs/subnet-evm/issues/500).

>We should update this so that if a commit or **branch name**

(emphasis mine)